stix_shifter_modules/splunk/stix_translation/json/from_stix_map.json

```diff
@@ -83,6 +83,7 @@
     "file": {
         "cim_type": "endpoint",
         "fields": {
+            "hashes": "file_hash",
```

Comment:
`file:hashes` is an incomplete STIX object path, and is invalid.

Reply:
Correct, this would need to be removed.

Reply:
please review it ,we have resolved this.